src/pieces/memberHandler.ts

```diff
@@ -35,6 +39,21 @@ async function memberUpdate(oldMember: GuildMember | PartialGuildMember, newMemb
 	}
 }
 
+async function memberRemove(member: GuildMember | PartialGuildMember): Promise<void> {
+	if (member.guild.id !== GUILDS.MAIN) return;
+
+	const dbMember: SageUser = await member.client.mongo.collection(DB.USERS).findOne({ discordId: member.id });
+
+	if (!dbMember) return;
+
+	dbMember.isVerified = false;
+	dbMember.discordId = '';
+	dbMember.roles = dbMember.roles.filter(role => role !== ROLES.VERIFIED && role !== ROLES.STAFF);
+	dbMember.isStaff = false;
```

Comment:
Would it be worth removing the levels of members who are removed/leave? This is considering that someone who is banned definitely won't rejoin, and those who are unverified/simply leave on their own probably will not rejoin (or are unable to due to the server not using traditional invite URLs)

Reply:
Hmmm... this is an interesting point, I'm willing to reset anyone who leaves' level to 1. @renross @bladetyr what do you think